content/kubernetes/deployment/deployment-options.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: Flexible deployment options
+linktitle: Deployment options
+description: Redis Enterprise for Kubernetes allows you to deploy to multiple namespaces. This article describes flexible deployment options you can use to meet your specific needs. 
+weight: 12
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/deployment/deployment-options.md,
+    /kubernetes/deployment/deployment-options/,
+]
+
+---
+Redis Enterprise for Kubernetes can be deployed in several different ways depending on your database needs.
+
+A single Redis Enterprise cluster (REC) can manage multiple Redis Enterprise database objects (REDB) in multiple namespaces within the same Kubernetes cluster.
+
+## Single REC and single namespace (one-to-one)
+
+The standard and simplest way is with your Redis Enterprise cluster (REC) and Redis Enterprise databases (REDB) residing within the same namespace. No additional configuration is required for this, since there is no communication required to cross namespaces. See [Deploy Redis Enterprise Software on Kubernetes]{{<relref "/kubernetes/deployment/quick-start.md">}}.
```

Comment:
```suggestion
The standard and simplest way is with your Redis Enterprise cluster (REC) and Redis Enterprise databases (REDB) residing within the same namespace. This requires no additional configuration since there is no communication required to cross namespaces. See [Deploy Redis Enterprise Software on Kubernetes]{{<relref "/kubernetes/deployment/quick-start.md">}}.
```